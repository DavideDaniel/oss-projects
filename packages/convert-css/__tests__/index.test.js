const cssModule = require('../index');

describe('convert-css module', () => {
  it('should match its snapshot', () => {
    expect(cssModule).toMatchSnapshot();
  });
});
