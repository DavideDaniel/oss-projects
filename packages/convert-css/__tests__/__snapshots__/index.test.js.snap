// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert-css module should match its snapshot 1`] = `
Object {
  "convertKey": [Function],
  "convertKeyNames": [Function],
  "cssToCamelizedJson": [Function],
  "cssToJson": [Function],
}
`;
